src/OrchardCore.Modules/OrchardCore.Localization/Views/AdminCulturePicker.cshtml

```diff
@@ -1,42 +1,38 @@
 @using OrchardCore.Localization
 @using OrchardCore.Environment.Shell
 
-@model AdminCulturePickerViewModel
```

Comment:
This might goes into another PR, the PR should fix the RTL issues, right?

Reply:
It could, but since @agriffard requested it, I am leaving it here. + Its an improvement to the theme itself which is what this PR is doing.